pkg/cmd/task/describe.go

```diff
@@ -68,9 +68,17 @@ No params
 {{- if eq (len .Task.Spec.Inputs.Params) 0 }}
 No params
 {{- else }}
-NAME	TYPE
+NAME	TYPE	DEFAULT_VALUE
```

Comment:
Can you remove _ and make it DEFAULT VALUE instead?

Reply:
@danielhelfand Thanks for pointing. PR updated with changes to pipeline describe also